### DISCUSSION TITLE
Empty the cache on MacOS

### DISCUSSION THREAD

**Discussion by bastrapp** [ente-io/ente#4110] [Q&A] [answered]
Hi, my cache directory <user>/Library/Application Support/ente holds over 10GB of data.

Is there a way to limit the amout of disk space this takes up or how do I empty it?

Thanks!

**Comment by mnvr** [comment, marked as answer]
* This is not the cache directory, it is the application's data (which indeed includes some caches too).
* There is no way currently to set a hard limit. That said, this shouldn't grow too big since we don't cache originals indefinitely.
* To empty it - (a) Uninstall the app, (b) Delete this directory, (c) Install the app again.

**Reply by bastrapp** [reply]
Thanks for the quick reply @mnvr
